### ISSUE TITLE
Accessibility issue with mobile menu

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#1383]
Feedback from DAC:
![image](https://user-images.githubusercontent.com/43404358/138479900-06d7676a-12b2-4031-b999-9120694cd3ba.png)
I’ve noticed a bug which is a high level fail 4.1.2 Name, Role, Value (A). When the page is magnified, if a user expands the Menu and Collapses it again, the aria-expanded attribute on the button toggles correctly. However, if the Menu button is expanded and then not explicitly collapsed, and a user then clicks to expand the ‘Search’, the Menu collapses, but the aria-expanded stays set to ‘true’. 

Steps to recreate:
1. Go to https://sdgdata.gov.uk/
2. Open up inspect console and resize so that mobile version of site is showing
3. Focus inspect on `<button data-target="menu-mobile" aria-expanded="false">Menu</button>`
4. Expand mobile menu - see that `aria-expanded` changes to true
5. Without collapsing the mobile menu, click on the 'Search' button to expand that.
6. Notice that `aria-expanded` stays as true even though the menu has been force closed by the expansion of mobile search
